### ISSUE TITLE
Proposal: Adjust ~/.earthly/config.yml via command line

### ISSUE THREAD

**Issue by vladaionescu** [earthly/earthly#675]
### Proposal

Similar to how `git` is able to adjust its config using commands like

```
git config --global user.email "MY_NAME@example.com"
```

Earthly should have a similar ability.

```
earthly config global.cache_size_mb 20000
earthly config git.github.com.auth https
earthly config git.github.com.user me
earthly config git.github.com.password '53cr37'
```

### Use-cases

* Configuring sandboxed CIs like GitHub Actions, Circle CI and Travis CI. It's easier to just have a few `earthly config` lines in the script than have to define checked-in config.json that is then copied in the right location during the build.
* Easier to share config settings in online snippets, without having to paste the entire config file

### Other notes

* The general formatting of the yml should not be changed (eg any comments should be preserved). Or alternatively, if comments are detected, output an error that the change cannot be made safely and the user can still go ahead and make the change with a `--force` flag, which eliminates the comments.

**Comment by dchw** [comment]
Resolved by #778

**Comment by vladaionescu** [comment]
This is now complete and GA'd, I believe. Marking as done :tada:
